### PR TITLE
Updating Readme.md with -> description

### PR DESCRIPTION
**Pull request by FSaurenbach**
with hyperlinks referencing the website of the plugin and updated the example with a newer link also made the text a bit better

### CODE REVIEW

**Comment by FSaurenbach** [comment]
@soywiz

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # korge-compose
 
-KProject adding support for JetPack compose composables to KorGE
+KProject adding support for [JetPack compose](https://www.jetbrains.com/lp/compose-mpp/) composables to KorGE and Korlibs
```
Comment (soywiz) [inline README.md]: I recently was aware that this should be Compose Multiplatform instead of JetPack Compose. Can you change it too?

Comment (FSaurenbach) [reply README.md]: Done

**Comment by soywiz** [comment]
Thanks!

**Pull request merged**